docs/manifest.json

```diff
@@ -184,46 +183,44 @@
     {
       "title": "Dotfiles",
       "description": "Learn how to personalize your workspace",
-      "icon_path": "./images/icons/art-pad.svg",
-      "path": "./dotfiles.md"
+      "path": "./dotfiles.md",
+      "icon_path": "./images/icons/art-pad.svg"
     },
     {
       "title": "Secrets",
-      "description": "Learn how to use secrets in your worskpace",
-      "icon_path": "./images/icons/secrets.svg",
-      "path": "./secrets.md"
+      "description": "Learn how to use secrets in your workspace",
+      "path": "./secrets.md",
+      "icon_path": "./images/icons/secrets.svg"
     },
     {
       "title": "Administration",
       "description": "How to install and deploy Coder",
-      "icon_path": "./images/icons/wrench.svg",
       "path": "./admin/index.md",
+      "icon_path": "./images/icons/wrench.svg",
       "children": [
         {
           "title": "Authentication",
           "description": "Learn how to set up authentication using GitHub or OpenID Connect",
-          "icon_path": "./images/icons/key.svg",
-          "path": "./admin/auth.md"
+          "path": "./admin/auth.md",
+          "icon_path": "./images/icons/key.svg"
         },
         {
           "title": "Users",
           "description": "Learn about user roles available in Coder and how to create and manage users",
-          "icon_path": "./images/icons/users.svg",
-          "path": "./admin/users.md"
+          "path": "./admin/users.md",
+          "icon_path": "./images/icons/users.svg"
         },
         {
           "title": "Groups",
           "description": "Learn how to manage user groups",
-          "icon_path": "./images/icons/group.svg",
           "path": "./admin/groups.md",
-          "state": "enterprise"
```

Comment:
This needs to be kept so that we can correctly indicate which features are enterprise-only.

Reply:
My bad, I missed this property. Thank you for raising it, Cian.